.github/workflows/branch.yml

```diff
@@ -89,4 +93,47 @@ jobs:
           options: '{ "conf": "debug" }'
           skip-tracking: false
           subdirectory: fastlane
-          bundle-install-path: CACHE_BUNDLER
\ No newline at end of file
+          bundle-install-path: CACHE_BUNDLER
+
+
+      - name: Ensure android directory exists
+        run: mkdir -p android
+
+      - name: Setup Gradle cache
+        uses: gradle/gradle-build-action@v2
+
+      - name: List root diretory contents
+        run: ls -la
+
+      - name: List android directory contents
+        run: ls -la android
```

Comment:
Do we need all these steps?

Reply:
Not really. I'll remove the root and android directory. I was using them to confirm gradle location